google/cloud/aiplatform/__init__.py

```diff
@@ -47,10 +48,24 @@
 """
 init = initializer.global_config.init
 
+log_params = metadata.metadata_service.log_params
+log_metrics = metadata.metadata_service.log_metrics
+set_experiment = metadata.metadata_service.set_experiment
+get_experiment = metadata.metadata_service.get_experiment
+set_run = metadata.metadata_service.set_run
```

Comment:
we can discuss more offline on this, but based on the initial feedback from Nate, it is better to just use `init` to change or set experiment and run, rather than having separated `set` methods. The init call can take just an experiment or run as input parameter, which should not reset the other parameters.

Reply:
We can talk offline. The init does that, takes experiment and run. And ends up calling set_experiment and set_run, underneath. We might not expose the latter ones. 
IMO we should. I basically prefer this:

```
aiplatform.init(experiment='my-experiment', run='first-run')
...
aiplatform.set_run(run='second-run')
```

instead of this:

```
aiplatform.init(experiment='my-experiment', run='first-run')
...
aiplatform.init(run='second-run')
```

The code is already in place, so I think we can keep it. Let's discuss tomorrow

Reply:
Discussed offline. We can get this in, and then un-expose if not needed